sessions/Session 20, 25th of October 2022.md.md

```diff
@@ -0,0 +1,69 @@
+#session #? October 25th 2022
+
+Recap - we are in [[Betrayers Rise]]
+
+Ozark & Herschel searched the Driders - 11 investigation
+
+Long bow, Longsword x2 Wysp doesn’t want these weapons so Herschel thinks he’s unwell or something is not right. It’s prob just trauma tbh.
+
+Stained glass needs opening - need to dance again? Kleion remembers the order to press and we try that or see if fire returns. Nathan offers to help with mage hand to press the order of symbols.
```

Comment:
`[[Kleion]]`

Reply:
> Initial review! I left an example of how I've been linking to the party members. We can define how we want to title each page. Since there are different contexts.

Can I/you merge then? 🙏 ❤️